BuyBitcoin/settings.py

```diff
@@ -169,7 +173,8 @@
 
 # Simplified static file serving.
 # https://warehouse.python.org/project/whitenoise/
-STATICFILES_STORAGE = 'whitenoise.storage.CompressedManifestStaticFilesStorage'
+# STATICFILES_STORAGE = 'whitenoise.storage.CompressedManifestStaticFilesStorage'
+WHITENOISE_ROOT = os.path.join(BASE_DIR, 'assets')
```

Comment:
does this still access middleware properly?

Reply:
How do I verify that?

Reply:
I think if you try python manage.py collectstatic and it exits without complaints you're good.